src/components/veileder-checkbox-liste/veileder-checkbox-liste.tsx

```diff
@@ -1,114 +1,61 @@
 import * as React from 'react';
-import {ReactNode, useEffect, useState} from 'react';
-import {connect} from 'react-redux';
+import {useEffect, useState} from 'react';
+import {useSelector} from 'react-redux';
 import AlertStripe from 'nav-frontend-alertstriper';
 import {Checkbox} from 'nav-frontend-skjema';
-import {endreFiltervalg} from '../../ducks/filtrering';
 import {VeiledereState} from '../../ducks/veiledere';
 import {FiltervalgModell, VeilederModell} from '../../model-interfaces';
 import './veileder-checkbox-liste.less';
-import {OversiktType} from '../../ducks/ui/listevisning';
+import {AppState} from '../../reducer';
+import NullstillValgKnapp from '../nullstill-valg-knapp/nullstill-valg-knapp';
 
-interface StateProps {
-    veiledere: VeiledereState;
-    filtervalg: FiltervalgModell;
-    veilederNavnQuery: string;
+interface VeilederCheckboxListeProps {
+    endreFiltervalg: (filterId: string, filterVerdi: any) => void;
 }
 
-interface DispatchProps {
-    sokEtterVeileder: (veiledere: string[]) => void;
-}
-
-interface KnappProps {
-    onClick: (e) => void;
-}
-
-const LukkeKnapp: React.SFC<KnappProps> = (props: KnappProps) => (
-    <button
-        className="knapp knapp--mini checkbox-liste__valg-knapp"
-        type="button"
-        onClick={props.onClick}
-        data-testid="veilederoversikt_sok-veileder_lukk-knapp"
-    >
-        Lukk
-    </button>
-);
-
-const SubmitKnapp: React.SFC<KnappProps> = (props: KnappProps) => (
-    <button
-        className="knapp knapp--mini knapp--hoved checkbox-liste__valg-knapp"
-        type="button"
-        onClick={props.onClick}
-        data-testid="veilederoversikt_sok-veileder_velg-knapp"
-    >
-        Velg
-    </button>
-);
-
-function VeilederCheckboxListe(props: any) {
+function VeilederCheckboxListe({endreFiltervalg}: VeilederCheckboxListeProps) {
+    const filtervalg: FiltervalgModell = useSelector((state: AppState) => state.filtreringVeilederoversikt);
+    const veiledere: VeiledereState = useSelector((state: AppState) => state.veiledere); //SAMME SOM VALG
+    const veilederNavnQuery = useSelector((state: AppState) => state.filtreringVeilederoversikt.veilederNavnQuery);
     const [valgteElementer, setValgteElementer] = useState<string[]>([]);
```

Comment:
maybe better to replace elementer with something more meaningful...

Reply:
well spotted :D